Assets/Scripts/Door.cs

```diff
@@ -35,9 +35,10 @@ private void OnTriggerExit2D(Collider2D collision)
 	void Update () {
         if (Input.GetButtonDown("Activate") && isPlayerInTrigger)
         {
+            DontDestroyOnLoad(this.gameObject);
```

Comment:
This will leave your door around. it's nice for making sure the sound plays, but now you have a door in the next level that you might not want...